### ISSUE TITLE
[Bug][a11y]: Swatches in documentation are missing a label

### ISSUE THREAD

**Issue by dineshy87** [adobe/spectrum-web-components#3320]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Swatch

### Expected behavior

All examples provided in documentation must have proper labels. Also, when an explicit `label` is not provided, it could fallback on the `color` value as it's aria-label. 

Once implemented, the documentation website must be updated in 'Accessibility' section with all the relevant information.

Note: Most components do not seem to have accessibility specific documentation

### Actual behavior

Currently, swatches are not labelled and there is no fallback option using color values.

### How can we reproduce this issue?

1. Go to 'https://opensource.adobe.com/spectrum-web-components/components/swatch/'
2.Run axe devtools to see the labelling errors

**Comment by TarunAdobe** [comment]
Hi, I checked the code and everything and I think the usecase that you've mentioned is already been handled as the aria-label is falling back to the color value when the label is not being provided.

**Comment by dineshy87** [comment]
[It was suggested ](https://github.com/adobe/spectrum-web-components/issues/3298) that this issue is created separately. But it looks like it was handled by a PR [#3330](https://github.com/adobe/spectrum-web-components/pull/3330/files)  that solved a different issue. If you verified that this works, then this can be closed.

**Comment by Rajdeepc** [comment]
@TarunAdobe Handled this in one of my PR in previous sprint which should fix this. Closing this as it is solved now. Thanks everyone for your inputs.
